### PR TITLE
add motoko course in an iframe

### PR DESCRIPTION
**Pull request by olaszakos**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/motoko/course.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Motoko Tutorial
```
Comment (jessiemongeon1) [inline docs/motoko/course.mdx:2]:
```suggestion
title: Motoko interactive tutorial
```

File: docs/motoko/tutorial.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Motoko Tutorial
```
Comment (jessiemongeon1) [inline docs/motoko/tutorial.mdx:2]:
```suggestion
title: Motoko interactive tutorial
```

File: docs/motoko/tutorial.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+hide_title: true
+hide_table_of_contents: true
+description: In the Motoko Tutorial, you will learn the basics of the Motoko programming language and how to use it to build programs on the Internet Computer.
```
Comment (jessiemongeon1) [inline docs/motoko/tutorial.mdx:5]:
```suggestion
description: In the Motoko interactive tutorial, you will learn the basics of the Motoko programming language and how to use it to build programs on the Internet Computer.
```

**Comment by olaszakos** [comment]
thank you @jessiemongeon1 for the review!

**Review by jessiemongeon1 — APPROVED** [review]

**Pull request merged**